packages/@aws-cdk/aws-eks/lib/fargate-cluster.ts

```diff
@@ -1,10 +1,18 @@
 import { Construct } from '@aws-cdk/core';
 import { Cluster, ClusterOptions, CoreDnsComputeType } from './cluster';
+import { FargateProfileOptions } from './fargate-profile';
 
 /**
  * Configuration props for EKS Fargate.
  */
 export interface FargateClusterProps extends ClusterOptions {
+  /**
+   * Fargate Profile to create along with the cluster.
+   *
+   * @default - A profile called "default" with 'default' and 'kube-system'
+   *            selectors will be created if this is left undefined.
+   */
+  readonly fargateProfile?: FargateProfileOptions;
```

Comment:
I wonder if this should be called “defaultProfile” instead (since this is a fargate cluster the “fargate” prefix is not needed and we usually reserve it to resource attributes (like the physical name))

Reply:
@eladb A case could probably be made for simply calling it `profile` too, I think. If leaving the value empty results in the same thing being created every time and having the name `default`, then what you'd be passing as options isn't really the default. That's why I went with `fargateProfile` instead of `defaultProfile`. What do you think?

Reply:
I updated the PR to use `defaultProfile` instead of `fargateProfile`.